go.mod

```diff
@@ -0,0 +1,3 @@
+module gopkg.in/urfave/fli.v2
```

Comment:
Fli? Also, why gopkg.in?

Reply:
@AudriusButkevicius ops, will fix that. What import path should it be? The README for v2 says use gopkg.in which is why I added that

Reply:
I guess this needs a wider discussion on how we deal with this going forward. With go.mod support I think gopkg.in is obsolete, and people just `go get github.com/urfave/cli@2.0.0`

Reply:
@AudriusButkevicius `github.com/urfave/cli@2.0.0` would not work. Per "Semantic Import Versioning" it would have to be `github.com/urfave/cli/v2` and similarly `go get github.com/urfave/cli/v2@v2.1.9` and so on. 

Note that Go Modules has a special handling of Gopkg.in so it should work okay. That said, I'm happy to remove the go.mod file from this PR and open a new issue for what the import path should be/

Reply:
Why would @2.0.0 not work, I was under the assumption that just resolves the release tag. Not sure why it needs to move to a separate subpackage to work?

Reply:
@AudriusButkevicius "Semantic Import Versioning" is a Module rule where it forces the major tag (2 and above) to be included in the import path. See here: https://github.com/golang/go/wiki/Modules#semantic-import-versioning

And this image is a good summary: 

![image](https://user-images.githubusercontent.com/16294261/62489443-bfa7d180-b794-11e9-89a4-c771e8b7b437.png)

Reply:
Ok, I am not sure how we'll handle this. 
Ideally I'd like to get away from gopkg.in, as we'll always live with master being v1, and go get github.com/urface/cli fetching v1.

Reply:
@AudriusButkevicius `go get github.com/urfave/cli` will *always* fetch v1, and `go get github.com/urfave/cli/v2` will always fetch v2 and so on.

Reply:
Use, unless we merge v2 to master and roll with that. I don't think two versions of the library should exist in the same repo basically.

Reply:
@AudriusButkevicius Go Modules does not care about what's in master, when you have semver tags. It will basically look at git tags and determine what version to get you.

Reply:
@AudriusButkevicius for this PR however, should I just remove the go.mod file? Or should I change it to be `github.com/urfave/cli/v2`?

Reply:
No idea, thats why I want someone else from @urfave/cli to comment and drive the direction of the migration.